packages/primitives/button/test/fixtures/bare-bones.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { Button } from '../../src'
+
+const App = () => (
```

Comment:
_Commenting in arbitrary place, so everyone can respond in thread_
I have couple of questions:
1. Does this approach mean that we will have to write a fixture for every combination of props we want to test?
2. Will we have to bring the same tests for some/all components in bubble-next?

Reply:
just for reference: we had a meeting to address those points.